lib/cli/lookup-files.js

```diff
@@ -59,31 +87,30 @@ const hasMatchingExtname = (pathname, exts = []) =>
  * @public
  * @alias module:lib/cli.lookupFiles
  * @param {string} filepath - Base path to start searching from.
- * @param {string[]} [extensions=[]] - File extensions to look for.
+ * @param {string[]} [fileExtensions=[]] - File extensions to look for.
  * @param {boolean} [recursive=false] - Whether to recurse into subdirectories.
- * @return {string[]} An array of paths.
+ * @returns {string[]} An array of paths.
  * @throws {Error} if no files match pattern.
- * @throws {TypeError} if `filepath` is directory and `extensions` not provided.
+ * @throws {TypeError} if `filepath` is directory and `fileExtensions` not 
+ * provided or an empty array.
  */
 module.exports = function lookupFiles(
   filepath,
-  extensions = [],
+  fileExtensions,
```

Comment:
let's make the code match the comment. I know `normalizeFileExtensions` does this for us but explicit is a bit more readable

```suggestion
  fileExtensions = [],
```